UIKitDZ/UIKitDZ/VC/StopwatchViewController.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  ViewController.swift
+//  UIKitDZ
+//
+//  Created by Анастасия Козлова on 21.09.2022.
+//
+
+import UIKit
+/// Экран секундомер
+final class StopwatchViewController: UIViewController {
+    
+    enum Constants {
+        static let space = "                                                        "
+        static let cellText = "Круг"
+        static let start = "Старт"
+        static let indentifire = "MyCell"
+        static let stop = "Стоп"
+        static let startTime = "00:00:00"
+        static let newTime = "Сброс"
+        static let defaultTime = "00"
+    }
+    
+    // MARK: - Visual Components
+    
+    let startButton: UIButton = {
+        
+        let button = UIButton()
+        button.setTitle(Constants.start, for: .normal)
+        button.setTitleColor(UIColor.green, for: .normal)
+        button.layer.cornerRadius = 55
+        button.layer.borderWidth = 5
+        button.layer.borderColor = UIColor.green.cgColor
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.backgroundColor = .tertiaryLabel
+        return button
+    }()
+    
+    let circleButton: UIButton = {
+        
+        let button = UIButton()
+        button.setTitle(Constants.cellText, for: .normal)
+        button.setTitleColor(UIColor.opaqueSeparator, for: .normal)
+        button.layer.cornerRadius = 55
+        button.layer.borderWidth = 5
+        button.layer.borderColor = UIColor.tertiaryLabel.cgColor
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.backgroundColor = .tertiaryLabel
+        button.isEnabled = false
+        return button
+    }()
+    
+    let timeTableView: UITableView = {
+        let tableView = UITableView()
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        return tableView
+    }()
+    
+    var timeLabel: UILabel = {
+        
+        let label = UILabel()
+        label.text = Constants.startTime
+        label.font = .systemFont(ofSize: 88)
+        label.textColor = .white
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    // MARK: - Public Properties
+    
+    let indentifire = Constants.indentifire
+    
+    var lapTime: [String] = []
+    
+    // MARK: - Private Properties
+    
+    private var timer = Timer()
+    
+    private var isTimerRunning = false
+    
+    private var counter = 0.00
+    
+    private var valueStart = false
```

Comment:
```suggestion
    private var iSvalueStart = false
```
is всегда перед булом пишем